docs/rabbitmq.config.example

```diff
@@ -246,6 +246,10 @@
    %%
    %% {memory_monitor_interval, 2500},
 
+   %% The total memory available can be calculated from the OS resources
+   %% - default option - or provided as a configuration parameter:
+   %% {total_memory_available_override_value, "5000MB"},
```

Comment:
@dcorbacho thanks for pointing that out. I should have read the entire file rather than just this diff.